Apps/WebClient/src/Server/Controllers/ConfigurationController.cs

```diff
@@ -45,7 +45,9 @@ public ConfigurationController(IConfigurationService service)
         [HttpGet]
         public Models.ExternalConfiguration Index()
         {
-            return this.configservice.GetConfiguration();
+            Models.ExternalConfiguration config = this.configservice.GetConfiguration();
+            config.WebClient.ClientIPAddress = this.HttpContext.Connection.RemoteIpAddress?.MapToIPv4().ToString();
```

Comment:
can this value be null?

Reply:
yes the IP can be null for a few reasons.